HeartbeatKit/src/index.js

```diff
@@ -31,13 +31,18 @@ var MessageType = {
 
 function constructor() {
     var self = this,
-        isInitialized = false,
+        isHBInit = false,
```

Comment:
i think it'll be clearer if we call it `isHBInitialized`, the way we are doing isClientInitialized and isServerInitialized